native/c/zshmem.hpp

```diff
@@ -167,4 +171,8 @@ inline void set_content_length(uint64_t content_len)
   msync(ZShared::instance()->region, sizeof(ZSharedRegion), MS_SYNC);
 }
 
+#ifdef SWIG
+}
+#endif
```

Comment:
This technically can be removed after the PR below is merged:
- https://github.com/zowe/zowe-native-proto/pull/518
-